### ISSUE TITLE
Implement the functionality of the hold_secs data element for Home Assistant service remote.send_command

### ISSUE THREAD

**Issue by solomongumbie** [postlund/pyatv#1720]
### Short feature/function description

Home Assistant has an option to specify how long a button is held with the remote.send_command service. When I attempt to use this with your Apple TV integration (which I love btw) it behaves just like a normal button push. I would like to be able to use this with the select command in order to be able to start the process of rearranging apps on the home screen. I know you have the option to send the home_hold command and a similar option with the select command would be very nice as well. However, if it were just as easy to implement the use of the hold_secs data element this could be useful for other applications too.

Example service YAML:
```
service: remote.send_command
data:
  hold_secs: 3
  command: select
target:
  entity_id: remote.bedroom_atv
```

### What needs to be done?

Allow for specifying of hold_secs data element or some other method of a select button hold command.

### Is this a breaking change?

No

### Anything else worth knowing?

I appreciate your project very much. This would be the last thing needed to complete my home made Harmony remote replacement system with no loss of functionality.

**Comment by postlund** [comment]
Yeah, this is on my bad-conscious-list for not implementing. It's basically the glue code in Home Assistant that is needed. I'm very swamped right now so I don't have much time for pyatv or Home Assistant development I'm afraid 😞 But it's nice to hear that you find the integration useful 😃

**Comment by solomongumbie** [comment]
I understand, thanks for all you have done already. Maybe sometime in the future.

**Comment by postlund** [comment]
It will definitely be implemented in the future. Maybe someone else gives it a spin? Should be fairly easy to implement.
